app/helpers/progress_report_helper.rb

```diff
@@ -1,13 +1,13 @@
 module ProgressReportHelper
   def is_progress_report_page?
-    controller.class == Teachers::ProgressReportsController || 
+    controller.class == Teachers::ProgressReportsController ||
     controller.class.parent == Teachers::ProgressReports ||
     controller.class.parent == Teachers::ProgressReports::Standards ||
     controller.class.parent == Teachers::ProgressReports::Concepts
   end
 
   def dont_show_premium_bar?
-    current_user.premium_state == 'premium'
+    current_user.premium_state == 'paid'
```

Comment:
Unrelated change, but 'premium' is not a value that can be returned by `premium_state`, so I changed it to 'paid'